### PR TITLE
Add options for volumeMounts and volumes in the Metrics Server chart

### PR DESCRIPTION
**Pull request by swade1987**
Signed-off-by: Steven Wade <steven@stevenwade.co.uk>

@olemarkus @kennethaasan

#### What this PR does / why we need it:

This PR allows users to specify any additional volumes or volume mounts they may require.

An example of the need for this is mounting secrets to be able add the following args:

```
- --tls-cert-file=/etc/kubernetes/secrets/apiserver.crt
- --tls-private-key-file=/etc/kubernetes/secrets/apiserver.key
```

#### Which issue this PR fixes
N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/metrics-server/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.3.1
 description: Metrics Server is a cluster-wide aggregator of resource usage data.
 name: metrics-server
-version: 2.3.0
+version: 2.3.1
```
Comment (olemarkus) [inline stable/metrics-server/Chart.yaml]: This needs a minor version bump

Comment (swade1987) [reply stable/metrics-server/Chart.yaml]: Fixed, bumped to 2.4.0

**Comment by swade1987** [comment]
@olemarkus think we are good now finally!

**Pull request merged**